### PR TITLE
Schemas for PAC, Next Video and V2 annotations

### PR DESCRIPTION
**Pull request by asparuhft**
# Description

## What

Schemas for PAC, Next Video, V2 annotations and constraints for predicates and concept types

## Why

JIRA Ticket [UPPSF-4370](https://financialtimes.atlassian.net/browse/UPPSF-4370)

## Anything, in particular, you'd like to highlight to reviewers

Mention here sections of code which you would like reviewers to pay extra attention to .E.g

_Would appreciate a second pair of eyes on the test_  
_I am not quite sure how this bit works_  
_Is there a better library for doing x_  

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [x] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

## DoD - Ensure all relevant tasks are completed before marking this PR as "Ready for review"

- [ ] Test coverage is not significantly decreased
- [ ] All PR checks have passed
- [ ] Changes are deployed on dev before asking for review
- [ ] Documentations remains up-to-date
  - [ ] OpenAPI definition file is updated
  - [ ] README file is updated
  - [ ] Documentation is updated in upp-docs and upp-public-docs
  - [ ] Architecture diagrams are updated

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

[UPPSF-4370]: https://financialtimes.atlassian.net/browse/UPPSF-4370?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: schemas/predicates.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (ivanruski) [inline schemas/predicates.json:1]: **question:** How is this file going to be used ?

**Review by ivanruski — APPROVED** [review]
LGTM.

File: schemas/predicates.json (hunk 1 above)
Comment (asparuhft) [reply schemas/predicates.json:1]: It is just a configuration file using JSON Schema format. It would require custom logic for the validation.

**Comment by asparuhft** [comment]
> **suggestion (non-blocking):** For the future versions of the ontology I would suggest to have more unified format of the predicate enum across the different annotation lifecycles. Also might be useful to have the lifecycle name in the schema as well, not only in the name of the file.
> 
> annotations-v2.json: **suggestion (non-blocking):** I would propose to have `prefLabel` and `types` as optional properties moving forward with the iterations of the annotations ontology.

Cool. I would address these suggestions with the next PR.

**Pull request merged**